### PR TITLE
Spinner component themeable migration

### PR DESCRIPTION
**Pull request by ke1k0**

### CODE REVIEW

File: packages/ui-spinner/src/Spinner/__tests__/Spinner.test.js
Comment (matyasf) [inline packages/ui-spinner/src/Spinner/__tests__/Spinner.test.js:60]: We should use common linting rules that are auto-applied at latest when committing code to prevent formatting-only changes.

File: packages/ui-view/src/View/index.js
Comment (matyasf) [inline packages/ui-view/src/View/index.js:64]: This is just a huge auto-formatting change?

File: packages/ui-view/src/View/index.js
Comment (ke1k0) [inline packages/ui-view/src/View/index.js:619]: @matyasf this is the only real change in the file, the rest is made by prettier "lint on save"

File: packages/ui-spinner/src/Spinner/__tests__/Spinner.test.js
Comment (ke1k0) [reply packages/ui-spinner/src/Spinner/__tests__/Spinner.test.js:60]: This change was made by prettier, we will continue to use auto linting on commit (and optionally in the IDE too).

File: packages/ui-view/src/View/index.js
Comment (ke1k0) [reply packages/ui-view/src/View/index.js:64]: Answered in https://github.com/instructure/instructure-ui/pull/157#discussion_r527562594.

File: packages/ui-spinner/src/Spinner/styles.js
Comment (matyasf) [reply packages/ui-spinner/src/Spinner/styles.js:142]: IMO it should be deleted, its just confusing

**Review by matyasf — APPROVED** [review]
Looks good, before merge we should decide what to do with the IE stuff

File: packages/ui-spinner/src/Spinner/styles.js
Comment (ke1k0) [reply packages/ui-spinner/src/Spinner/styles.js]: Removed the IE11 condition.

**Review by Brailor — APPROVED** [review]

**Pull request merged**
